app/views/school/welcome_wizard/la_funded_devices_you_can_order.html.erb

```diff
@@ -0,0 +1,50 @@
+<%- title = t('page_titles.school_user_welcome_wizard.range_of_devices.title') %>
+<%- content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <%= form_for @wizard, url: welcome_wizard_school_path(step: @wizard.step) do |f| %>
+      <%= f.hidden_field :step %>
+
+      <h1 class="govuk-heading-xl">
+        You can order a range of devices
+      </h1>
+
+      <p class="govuk-body">
+        We recommend checking with independent settings as soon as possible to find out which device type
+        and IT settings would be most suitable for each pupil.
+      </p>
+
+      <p class="govuk-body">
+        Depending on availability, you can choose from:
+      </p>
+
+      <h2 class="govuk-heading-s">
+        Standard Windows laptops
+      </h2>
+      <p class="govuk-body">
+        You’ll be responsible for arranging and maintaining safeguarding measures, like content filtering.
+      </p>
+      <h2 class="govuk-heading-s">
+        DfE Restricted Windows laptops
+      </h2>
+
+      <p class="govuk-body">
+        These have temporary content filtering and mobile device management software installed, which you’ll
+        need to replace when they expire. You’ll need to reset these devices before you can apply new
+        settings, or download some types of software.
+      </p>
+      <h2 class="govuk-heading-s">
+        Google Chromebooks
+      </h2>
+      <p class="govuk-body">
+        You’ll need to provide us with the relevant school’s email domain for <span class="app-no-wrap">G
+            Suite for Education</span>, and a recovery email address.
+      </p>
+      <p class="govuk_link_to"><a href="/devices/device-specification">See detailed laptop specifications</a></p>
```

Comment:
The `p` tag should be `govuk-body` and the link should be rendered with the method `govuk_link_to`.